project-p1-first-webpage-master/code/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New Zwischenziel 🏨 🏔</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <menu>
+      <a href="./projekte.html">Projekte</a>
+    </menu>
+    <!-- dont forget to add a css file and link it here! -->
+    <!-- If you're using google fonts, that should be added here -->
+    <img class="hero" src="../images/Zwischenziel.png" alt="Zwischenziel" />
+  </head>
+  <body>
+    <p>
+      „zwischenziel“ ist ein verein, der leerstehende gastro- und hotelflächen
+      entlang der nationalstrasse n11 zur zwischennutzung an interessierte
+      personen oder gruppen zur befristeten weitermiete vermittelt.
+      <br />
+      <br />
+      im fokus steht die förderung und ermöglichung von erfolgreichen
+      zwischennutzungen mit positiven effekten auf die gesellschaft, wirtschaft
+      und kultur. der verein ist nicht gewinnorientiert. allfällige
+      überschüssige gelder werden in weitere projekte investiert.
+      <br />
+      <br />
+    </p>
+    <br>
+    <br>
```

Comment:
Try setting some margin instead of line breaks if you want to add some whitespace in your page 👍